driver-core/src/main/java/com/datastax/driver/core/ConfigurationBundle.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) 2020 ScyllaDB
+ */
+
+package com.datastax.driver.core;
+
+import com.google.common.base.Throwables;
+import java.io.*;
+import java.security.GeneralSecurityException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.TrustManagerFactory;
+import javax.net.ssl.X509TrustManager;
+
+public class ConfigurationBundle {
```

Comment:
```suggestion
class ConfigurationBundle {
```

Let's make it private (if possible). The reason is that we can more easily modify that in the future without worrying about backwards compatiblity or someone manually using `ConfigurationBundle` which is an object for internal use only.